visualization/include/pcl/visualization/impl/point_cloud_color_handlers.hpp

```diff
@@ -174,7 +174,6 @@ PointCloudColorHandlerRGBField<PointT>::getColor () const
           !std::isfinite ((*cloud_)[cp].y) ||
           !std::isfinite ((*cloud_)[cp].z))
         continue;
-
       memcpy (&rgb, (reinterpret_cast<const char *> (&(*cloud_)[cp])) + rgba_offset, sizeof (pcl::RGB));
```

Comment:
memcpy untouched?

Reply:
Incompatible types? I left alone the `memcpy` that were stuffing some kind of `PointT` data into `pcl::RGB`